src/source/Sctp/Sctp.c

```diff
@@ -199,6 +199,9 @@ STATUS sctpSessionWriteDcep(PSctpSession pSctpSession, UINT32 streamId, PCHAR pC
      pPacket[0] = DCEP_DATA_CHANNEL_OPEN;
      pPacket[1] = DCEP_DATA_CHANNEL_RELIABLE;
      MEMCPY(pPacket + 8, &channelLenNetworkOrder, SIZEOF(UINT16));
+     //TODO: proto len
```

Comment:
Not sure what the comments are telling. Please add more context or remove.

Ideally, I would like to have #define the offsets here. Also, instead of ntos the channel name len, we should use unaligned be macro directly